README.md

```diff
@@ -81,7 +81,9 @@ require 'pry'
 
 # Install Puppet on all hosts
 hosts.each do |host|
-  on host, install_puppet
+  # Select the correct method below.
+  on host, install_puppet  # this is for Puppet 3
+  install_puppet_agent_on(host, :version => '1.5.2') # this is for Puppet 4
```

Comment:
yes, it is. There's one known issue with using `install_puppet_on`, however, which can be gotten around by providing the global options hash as the `opts` parameter to that method.